spec/decorators/punch_decorator_spec.rb

```diff
@@ -2,4 +2,21 @@
 
 describe PunchDecorator do
 
+  describe "#extra_hour as 'no'" do 
+    let(:punch) { create (:punch) }
+    let(:punch_decorator) { PunchDecorator.new(punch) }
+
+    it "translates the bool value into 'no'" do 
+      expect(punch_decorator.extra_hour).to eq("#{I18n.t(false)}")
+    end 
+  end 
+
+  describe "#extra_hour as 'yes'" do 
+    let(:punch) { create :punch, :is_extra_hour }
+    let(:punch_decorator) { PunchDecorator.new(punch) }
+
+    it "translates the bool value into 'yes'" do 
+      expect(punch_decorator.extra_hour).to eq("#{I18n.t(true)}")
```

Comment:
here you should check the text ` expect(punch_decorator.extra_hour).to eq("yes")`

if for some reason, someone changes the translation text from yes to another text this spec should break, if you check against the text from the translation `I18n.t(true)` will give an false positive

Reply:
Oh, that's so true!